src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php

```diff
@@ -122,6 +122,17 @@ public function process(ContainerBuilder $container)
             ExtractingEventDispatcher::$aliases = [];
         }
     }
+
+    private function getDefaultPriority(string $id, ContainerBuilder $container): int
+    {
+        $class = $container->getDefinition($id)->getClass();
+
+        if (null !== $class && ($r = $container->getReflectionClass($class, false)) && $r->hasMethod('getDefaultPriority')) {
```

Comment:
Should I check for an Interface (_Does not exist yet_) here instead of the `$r->hasMethod('getDefaultPriority')`?